cmd/kubelet/app/options/globalflags.go

```diff
@@ -93,7 +93,21 @@ func addCredentialProviderFlags(fs *pflag.FlagSet) {
 
 // addKlogFlags adds flags from k8s.io/klog
 func addKlogFlags(fs *pflag.FlagSet) {
```

Comment:
can we add a test for this?

Reply:
That's a good idea. Will have to think about how.

Reply:
Probably just call Set manually for one of the flags, with a known non-default value, then construct a new FlagSet and extract the value to see if it still matches what we set. Don't have time to try today but I'll give it a shot tomorrow.

Reply:
Added tests.